packages/engine/Specs/Scene/PrimitiveCullingSpec.js

```diff
@@ -158,12 +157,7 @@ describe(
       // render until all labels have been updated
       return pollToPromise(function () {
         scene.renderForSpecs();
-        const backgroundBillboard =
-          labels._backgroundBillboardCollection.get(0);
-        return (
-          (!defined(backgroundBillboard) || backgroundBillboard.ready) &&
-          labels._labelsToUpdate.length === 0
-        );
```

Comment:
same question , is this related to the texture refactor? (again just for my own understanding...)

Reply:
This one is related. I exposed a `ready` property which covers this.